packages/contracts/test/contracts/OVM/bridge/base/OVM_L1CrossDomainMessenger.spec.ts

```diff
@@ -355,12 +374,8 @@ describe('OVM_L1CrossDomainMessenger', () => {
       ).to.be.revertedWith('Provided message has already been received.')
     })
 
-    it('when the OVM_L2MessageRelayer address is set, should revert if called by a different account', async () => {
```

Comment:
WIth https://github.com/ethereum-optimism/optimism/issues/623 in mind as being separated out, can we keep this test for the time being?

Reply:
This test case is still here, just got moved down. I think git lost track of the diff because I also wrapped it in a new `describe` to organize the different `relayMessage` tests a bit more cleanly.